### ISSUE TITLE
Response [403] Forbidden

### ISSUE THREAD

**Issue by chaitanya2ivvala** [sat-utils/sat-search#94]
I am not sure why this error occurred i am facing this issue since morning when i am trying to use sat-search module. 

<Response [403]>
Traceback (most recent call last):
  File "funcSearch.py", line 736, in <module>
    handler(newjson, "Alex")
  File "funcSearch.py", line 594, in handler
    listfile, listtime = searchcomputing(satellitetype, inputjson, startday, timerange, endday, cloudfrom, cloudto)
  File "funcSearch.py", line 105, in searchcomputing
    listMRGS = search.GetMGRSpositionOverlap(inputjson)
  File "/home/chaitanya/vultus/platform_core/adapters/search/docker/search.py", line 84, in GetMGRSpositionOverlap
    items1 = search.items()
  File "/home/chaitanya/vultus/sample_environment/lib/python3.6/site-packages/satsearch/search.py", line 118, in items
    found = self.found()
  File "/home/chaitanya/vultus/sample_environment/lib/python3.6/site-packages/satsearch/search.py", line 75, in found
    results = self.query(**kwargs)
  File "/home/chaitanya/vultus/sample_environment/lib/python3.6/site-packages/satsearch/search.py", line 86, in query
    raise SatSearchError(response.text)
satsearch.search.SatSearchError: {"message":"Forbidden"}

**Comment by matthewhanson** [comment]
Hello @chaitanya2ivvala , sorry that we broke this.
The problem lies in the DNS for the default STAC API that sat-search uses. We had to update the DNS to point to the new STAC API, which broke the old one. 

To fix this on your end, just set the environment variable to the actual URL:
SATUTILS_API_URL=https://hb57pr2zof.execute-api.us-east-1.amazonaws.com/stage/

And it will work as it did before.

Note that there is a new API now using STAC 1.0.0-beta.1, but it does not yet contain Landsat or CBERS, it has Sentinel-2 L1C, L2A, and newly created L2A COGs for Africa.   It is located at:
https://earth-search.aws.element84.com/v0

There is a new sat-search in release candidate form, and the final release will happen in the next couple days.
